src/components/SideNav.js

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import { GrFormClose } from 'react-icons/gr';
 import { Link } from 'react-router-dom';
 import LOGO from '../images/Tour-Hunter.png';
 import SocialIcons from './SocialIcons';
 
 const SideNav = () => {
+  const signedIn = useSelector((state) => state.signedIn);
+
   const Links = [
     { id: 'Home', src: 'Home', path: '/tours' },
     { id: 'Reservation', src: 'My Reservations', path: '/reservations' },
     { id: 'new-tour', src: 'New Tour', path: '/tour/new' },
     { id: 'delete', src: 'Delete Tour', path: '/tours/delete' },
-    { id: 'signout', src: 'Sign Out', path: '/' },
+    {
+      id: 'signout',
+      src: signedIn === 'Not signed in' ? 'Sign up' : 'Sign in',
+      path: signedIn === 'Not signed in' ? '/signup' : '/',
+    },
+    { id: 'signin', src: 'Sign In', path: '/users/sign_in' },
```

Comment:
**Optional change:

- [ ]  When the user signs in, in the Sidenav, the options to sign up and sign in still appear. I think it could be better if logged users only can see the sign-out option in the Sidenav.